main.go

```diff
@@ -131,19 +131,21 @@ func main() {
 					Usage: "generate a migration that will not be wrapped in a transaction when run",
 				},
 			},
-			Action: func(c *cli.Context) {
+			Action: func(c *cli.Context) error {
 				if len(c.Args()) == 0 {
-					println("migration name not given! try `pgmgr migration NameGoesHere`")
-				} else {
-					displayErrorOrMessage(pgmgr.CreateMigration(config, c.Args()[0], c.Bool("no-txn")))
+					fmt.Println("migration name not given! try `pgmgr migration NameGoesHere`")
```

Comment:
This probably ought to be returned as a cli.NewExitError too, since it's an error case.